src/communication/websocket/session.cpp

```diff
@@ -80,6 +80,7 @@ bool Session::Run() {
     return false;
   }
 
+  authentication_process_ = auth_.HasAnyUsers();
```

Comment:
You can just use `authenticated_ = !auth_.HasAnyUsers()` here.
And then the function `IsAuthenticated` can just be `return authenticated_`.

Reply:
Aha that is true, it makes everything simpler